test/Performance/README.md

```diff
@@ -28,13 +27,13 @@ On Linux and Mac, the script must be executed with `sudo python3` because to ena
     
     python3 run-perf_tests.py -platform <PLATFORM> -iterations <ITERATIONS> -iterations-large <ITERATIONS_LARGE> -result-server <RESULT_SERVER> -result-db <RESULT_DB> -result-uid <RESULT_UID> -result-pwd <RESULT_PWD>
 
-`-platform` - The platform that the tests are ran on. Must be one of the following: Windows10, WindowsServer2016 WindowsServer2012 Ubuntu16 RedHat7 Sierra  
+`-platform` - The platform that the tests are ran on. Must be one of the following: Windows10, WindowsServer2016, WindowsServer2012, Ubuntu16, RedHat7, Sierra  .
 `-iterations` - The number of iterations for regular tests.  
 `-iterations-large` - The number of iterations for the tests that fetch large data. Usually set to 1.  
-`-result-server` - The server of result database. It is assumed that, the result database already setup before running the tests.  
-`-result-db` - Database name. With the current result database setup files, this should be set to `TestResults`  
-`-result-uid` - Result database username  
-`-result-pwd` Result database password  
+`-result-server` - The server containing the result database. It is assumed that the result database s already setup before running the tests.  

```

Comment:
It is assumed that the result database is already setup before running the tests (I guess)